go/vt/sqlparser/sql.y

```diff
@@ -348,7 +361,7 @@ func skipToEnd(yylex interface{}) {
 %type <strs> enum_values
 %type <columnDefinition> column_definition column_definition_for_create
 %type <indexDefinition> index_definition
-%type <constraintDefinition> constraint_definition, check_constraint_definition
+%type <constraintDefinition> constraint_definition check_constraint_definition
```

Comment:
Did this even work with the comma? apparently?

Reply:
It did, but the linter says it shouldn't lol, so I just removed them